### PR TITLE
Add seccomp errno filter support

### PR DESCRIPTION
**Pull request by wader**
No tests yet. Tried to get the existing tests running without much success :(
Would users like to mix errno and black/white -lists? i currently only need errno filter and default allow.
Tried to keep same code style, hope it is ok.

### CODE REVIEW

File: src/firejail/seccomp.c [hunk 1]
```diff
@@ -157,6 +161,11 @@ void filter_debug(void) {
 			printf("  BLACKLIST %d %s\n", *nr, syscall_find_nr(*nr));
 			i += 2;
 		}
+		else if (*ptr	== 0x15 && *(ptr +14) == 0x5 && *(ptr + 15) == 0) {
```
Comment (wader) [inline src/firejail/seccomp.c]: maybe this function could be rewritten using sock_filter struct pointer instead hmm would also take care of the endian stuff

**Comment by wader** [comment]
For some context, i myself use this to workaround buggy virtualbox shared filesystem that has broken sendfile support, the application will fallback to userland buffered IO if it gets ENOTSUP.

**Comment by netblue30** [comment]
Looks fine, I'll merge it over the weekend. Thanks!

**Comment by wader** [comment]
Ok. Please run the tests to make sure it didn't break anything. Any pointers how to run the tests? i installed expect and messed around but did not really work hmm

**Comment by netblue30** [comment]
I go in test directory and run:
`````
sudo ./configure
`````

Take a look at configure script first, it creates a number of bridge devices. To get rid of them I usually reboot my computer after the test.

I start the test as "./test.sh | grep TESTING"

**Pull request merged**